classes/helper/datafield_manager.php

```diff
@@ -93,6 +93,8 @@ public function update_datafields($event, $stepresults) {
         }
 
         $newfields["wwwroot"] = $CFG->wwwroot;
+        $url = new \moodle_url($CFG->wwwroot);
+        $newfields["wwwroot_domain"] = $url->get_host();
```

Comment:
This is not actually correct because a wwwroot could have a subdir:

https://www.example.com/sub/dir/  => "www.example.com/sub/dir/"

Very rarely it could also have a port number too.

https://www.example.com:1234/sub/dir/ -> www.example.com:1234/sub/dir/

So either add those back in, or perhaps it is easier to simply start with the wwwroot and then remove the scheme